extension/jaegerremotesampling/constants.go

```diff
@@ -0,0 +1,41 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
if this is a copy from Jaeger it should preserve Jaeger copyright header

Reply:
These licence headers break the checklicense lint
```go
// Copyright (c) 2018 The Jaeger Authors.
// SPDX-License-Identifier: Apache-2.0
```
Should it be changed to
```go
// Copyright (c) 2018 The Jaeger Authors.
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0
```

Reply:
no, the newer copyright line goes at the top
```
// Copyright The OpenTelemetry Authors
// Copyright (c) 2018 The Jaeger Authors.
// SPDX-License-Identifier: Apache-2.0
```